minimalTest.js

```diff
@@ -136,7 +136,7 @@ const messageData = {
     const view = document.getElementById("view");
     if (!this.replyingToMessage) {
       // 70px for reply box above the message input
-      view.style.height = `${view.offsetHeight - 70}px`;
+      // view.style.height = `${view.offsetHeight - 70}px`;
```

Comment:
lets just delete all these instead of commenting them out?

Reply:
removed in https://github.com/igneous-labs/omnispace/pull/71